addons/web/static/src/webclient/settings_form_view/settings_form_controller.js

```diff
@@ -110,6 +110,7 @@ export class SettingsFormController extends formView.Controller {
             clickParams: {
                 name: "cancel",
                 type: "object",
+                clearBreadcrumbs: true,
```

Comment:
I think it's rather `{ stackPosition: 'replaceCurrentAction' }`, because theoretically, the settings action might not be the first one in the breadcrumbs, and clearBreadcrumbs would remove everything.

Reply:
also, this should probably not be a clickParams. What about adding a second argument `options` to onClickViewButton, and propagating it to the call to doAction?

Reply:
Should I allow other options besides `stackPosition` ? What about `clearBreadcrumbs` ?

Reply:
you don't need to encode them, you can simply propagate the `options` object